### PR TITLE
adding a file for Monica

### PR DESCRIPTION
**Pull request by mdubrey**
created a function within the original code to be added to the branch

### CODE REVIEW

File: HelloWorldMonica.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    
+    public static void main(String[] args) {
+       
+        System.out.println("\nHello World!");
+        System.out.println("\nClass Collaboration Begins" );
+        monicaFunction();
+        // \n adds a new line
+    }
+    
+    public static void monicaFunction()
```
Comment (mdubrey) [inline HelloWorldMonica.java]: this is the monicaFunction() that was defined

**Comment by mdubrey** [comment]
It will be interesting to see how the merge works
@kwestberry
@thevillianinsideyou

**Comment by mdubrey** [comment]
Hey guys please review and comment before I merge this file.

**Comment by kwestberry** [comment]
Looks good to me lol.

**Comment by thevillianinsideyou** [comment]
Sounds good thanks for all the work you've contributed

**Pull request merged**